geoportal/c2cgeoportal_geoportal/scaffolds/create/build_tmpl

```diff
@@ -85,7 +85,7 @@ if auto or args.geoportal:
 
 if args.env or auto and not os.path.exists(".env"):
     with open(".env", "w") as dest:
-        env_files = args.env_files if args.env_files else [".env.sample"]
+        env_files = args.env_files if args.env_files else ["env.sample"]
```

Comment:
args.env_files already have a default value in parser.

Reply:
+1 thanks :-)